packages/inter-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -174,20 +176,19 @@ export const prepareVaultManagerKit = (
 
   const { priceAuthority, timerService, reservePublicFacet } = zcf.getTerms();
 
-  const makeVault = prepareVault(baggage, marshaller, zcf);
-  const makeVaultManagerPublishKit = prepareDurablePublishKit(
-    baggage,
-    'Vault Manager publish kit',
-  );
+  const makeVault = prepareVault(baggage, makeRecorderKit, zcf);
+
   const periodNotifier = E(timerService).makeNotifier(
     0n,
     factoryPowers.getGovernedParams().getChargingPeriod(),
   );
 
-  /** @type {PublishKit<AssetState>} */
-  const { publisher: assetPublisher, subscriber: assetSubscriber } =
-    makeVaultManagerPublishKit();
-  pipeTopicToStorage(assetSubscriber, storageNode, marshaller);
+  const assetKit = makeERecorderKit(
+    storageNode,
```

Comment:
Isn't `storageNode` a presence now in this case, which would allow using `makeRecorderKit`?

Reply:
Good catch! Will push that change